### DISCUSSION TITLE
Project Logo

### DISCUSSION THREAD

**Discussion by aahnik** [ramnes/notion-sdk-py#34] [Ideas]
This project desperately needs a logo. Humans are aesthetic beings. I am not a designer, but I imagine that a blend of python and notion logos would be suitable.

But how to blend them? I have an idea. Put python on the roof of Notion. But I don't know how to do it.

Let this thread be dedicated to project logo proposals by interested people.

**Comment by Daveismus** [comment]
So I am not entirly sure what was your idea, but i made some rough sketches. Once there is some resonance i can refine the most liked one.

# 1. 
<img src="https://user-images.githubusercontent.com/55088389/119983177-55869f00-bfaf-11eb-9899-19541ead6a07.png" width="200">

# 2.
<img src="https://user-images.githubusercontent.com/55088389/119983181-561f3580-bfaf-11eb-9204-7467959911d7.png" width="200">

# 3. 
<img src="https://user-images.githubusercontent.com/55088389/119983184-57506280-bfaf-11eb-8aff-59d83e2d6315.png" width="200">

# 4.
<img src="https://user-images.githubusercontent.com/55088389/119983185-57506280-bfaf-11eb-9236-e80c907fc32d.png" width="200">

My personal favorite is number 4

**Reply by aahnik** [reply]
Out of these four, I vote for no 4. I would love to know what @ramnes  thinks.

Your designs a wonderful!

**Reply by Blucknote** [reply]
On dark GitHub theme it's almost invisible
![image](https://user-images.githubusercontent.com/15163043/120068904-bbe0ef80-c08b-11eb-9632-527beffbde77.png)

**Reply by Daveismus** [reply]
I can swap from transparent background to white background to solve this issue. 
How is Notion solving this on there site?

**Reply by Blucknote** [reply]
I don't think this necessary. Bottom logo looks pretty good. I guess Notion solves this with same method: white background

**Comment by ramnes** [comment]
What about something like this?

![notion-sdk-py-logo1](https://user-images.githubusercontent.com/835072/120043944-269f1600-c00d-11eb-8b38-a81f5d2bb298.png)

**Reply by aahnik** [reply]
Beautiful illustration. The snake looks little scary. But is allright.

**Reply by Daveismus** [reply]
I See it like aahnik, a Bit scary, but a beautiful illustration

**Reply by Blucknote** [reply]
Whoa. It it protecting Notion from someone? :D

**Reply by aahnik** [reply]
it would have been great if the expression of the snake was a little benign

**Reply by ramnes** [reply]
Indeed a friendlier looking snake would be more welcoming! Alas I didn't draw the snake myself, it's a random illustration I found with Google Images. I'd be ready to give a few bucks for an illustrator to redraw this if you guys know anyone that can draw something in that style and make it an svg.
